src/app/payment/paymentMethod/WalletButtonPaymentMethod.tsx

```diff
@@ -281,22 +294,28 @@ function mapFromCheckoutProps(
     { checkoutService, checkoutState }: CheckoutContextProps,
     { method }: WalletButtonPaymentMethodProps
 ): WithCheckoutWalletButtonPaymentMethodProps | null {
-    const { data: { getBillingAddress, getCheckout } } = checkoutState;
+    const { data: { getBillingAddress, getCheckout, getCustomer } } = checkoutState;
```

Comment:
I think this is all you need:
```suggestion
    const { data: { getBillingAddress, getCheckout, isPaymentDataRequired } } = checkoutState;
```